console/server/src/main/java/quarks/console/server/ServerUtil.java

```diff
@@ -45,31 +53,35 @@ private String getPath() {
      * @return a File object
      */
     private File getTopDirFilePath() {
+        String topDirProp = System.getProperty("top.dir.file.path");
```

Comment:
That is probably a good idea. Do we want to wait until a new name is settled upon? Or just plan to change the name from quarks later?